web/src/css/flowdetail.less

```diff
@@ -41,6 +42,24 @@
 
 }
 
+.flowview-container{
+    display: flex;
+    flex-direction: column;
+    padding: 0 !important;
+}
+
+.flowview-body{
```

Comment:
It feels like this could just be `section.request, section.response` inserted at line 14/15.